src/persistentstorage.cpp

```diff
@@ -258,3 +266,229 @@ void MysqlPersistentStorage::writelog(const std::string &id, const ObjectValue &
     }
 }
 #endif // HAVE_MYSQL
+
+
+#ifdef HAVE_INFLUXDB
+Logger& InfluxdbPersistentStorage::logger_m(Logger::getInstance("InfluxdbPersistentStorage"));
+
+bool _existsInArray(const Json::Value &json, const std::string &key)
+{
+    bool ret = false;
+    assert(json.isArray());
+    for (int i = 0; i != json.size(); i++)
+    {
+        if (json[i][0].asString() == key)
+            ret = true;
```

Comment:
@fraxinas  Suggestions to simplify this method:
- This line can be replaced with
```
{
        return true;`
}
```
- delete the `bool ret` local var
- return `false` at the end of the method